test/relayed_owned_set.js

```diff
@@ -411,28 +387,26 @@ contract("TestRelayedOwnedSet", accounts => {
   it("should allow only the relayedOwned contract to call `initiateChange`", async () => {
     const relayedOwnedSetAddress = (await relayedOwnedSet()).address;
     const relay = await relaySet();
-    let watcher = relay.NewRelayed();
-
+    const kkk = await relay.relayedSet();
```

Comment:
```suggestion
    const relayedSet = await relay.relayedSet();
```

Reply:
this constant is not in use, you can just delete this line.